crates/bonsaidb-core/src/networking.rs

```diff
@@ -46,6 +48,7 @@ pub enum Request<T> {
         feature = "actionable-traits",
         actionable(protection = "none", subaction)
     )]
+    #[derive_where(skip_inner(Debug))]
```

Comment:
You could just write `#[derive_where(skip_inner)]` too, considering you are only implementing `Debug`. But I think explicitly specifying `Debug` is actually better? Do you have any thoughts on this?